### PR TITLE
Modified Curve's metapool-related keys

### PR DESCRIPTION
**Pull request by mariogutval**
Renamed keys in contracts.ts according to the Curve adapter contracts' names.

#### PR Checklist

- [ ] Tests

### CODE REVIEW

File: src/ethereum/curve/contracts.ts [hunk 1]
```diff
@@ -4881,7 +4881,7 @@ const contracts = {
       ],
     },
   },
-  CurveMetapoolSwapPool: {
+  CurveMetapoolSwapAdapter: {
```
Comment (deepcrazy) [inline src/ethereum/curve/contracts.ts]: Small doubt: Is `Adapter` suffix is required? If so, then For other existing keys like `CurveDepositPool`, It is not same like Adapter name [here](https://github.com/Opty-Fi/team-defi-adapters/blob/master/contracts/1_ethereum/curve/CurveDepositPoolAdapter.sol#L38)

**Review by deepcrazy — APPROVED** [review]
Looks fine. But have a small doubt.

File: src/ethereum/curve/contracts.ts (hunk 1 above)
Comment (leodinh) [reply src/ethereum/curve/contracts.ts]: Same doubt

Comment (dhruvinparikh) [reply src/ethereum/curve/contracts.ts]: - [ ] adapter is not required

**Review by dhruvinparikh — CHANGES_REQUESTED** [review]

File: src/ethereum/curve/contracts.ts [hunk 2]
```diff
@@ -1699,4781 +6477,7 @@ const contracts = {
...
-      tokens: ["0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-    "dsu+3crv": {
-      pool: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      lpToken: "0x6eC80df362d7042c50D4469bcfbc174C9dd9109a",
-      tokens: ["0x605D26FBd5be761089281d5cec2Ce86eeA667109", "0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490"],
-    },
-  },
-  CurveMetapoolGaugePool: {
+  CurveMetapoolGauge: {
```
Comment (deepcrazy) [inline src/ethereum/curve/contracts.ts:6480]: Small Doubt: Don't we need to keep pools under `pools` property here? Ot it is fine if we keep pools directly under `CurveMetapoolGauge` key?

**Review by deepcrazy — APPROVED** [review]
Approving. However have a small doubt. Please check.

File: src/ethereum/curve/contracts.ts (hunk 2 above)
Comment (mariogutval) [reply src/ethereum/curve/contracts.ts:6480]: I didn't add subkeys because the CurveMetapoolGauge ABI is the same as CurveGauge ABI. Does it make sense?

**Review by leodinh — APPROVED** [review]

File: src/ethereum/curve/contracts.ts (hunk 2 above)
Comment (deepcrazy) [reply src/ethereum/curve/contracts.ts:6480]: Discussed!

**Pull request merged**